app/services/agenda-service.js

```diff
@@ -128,11 +129,15 @@ export default class AgendaService extends Service {
     const defaultDecisionResultCodeUri = isAnnouncement ? CONSTANTS.DECISION_RESULT_CODE_URIS.KENNISNAME : CONSTANTS.DECISION_RESULT_CODE_URIS.GOEDGEKEURD;
     const decisionResultCode = await this.store.findRecordByUri('concept', defaultDecisionResultCodeUri);
 
+    // default secretary
+    const currentApplicationSecretary = await this.mandatees.getCurrentApplicationSecretary();
+
     // decision-activity
     const decisionActivity = await this.store.createRecord('decision-activity', {
       subcase,
       startDate: meeting.plannedStart,
       decisionResultCode,
+      secretary: currentApplicationSecretary
```

Comment:
We actually want to check whether the `meeting` has a `secretary` set, that might differ from the `currentApplicationSecretary`. Then we'd use the `currentApplicationSecretary` if the `meeting.secretary` is null (which it can't be for meetings created after this change, but we want to cover this edge case for older agendas).